src/Protector.php

```diff
@@ -368,7 +369,10 @@ protected function generateDump(string $destinationFilePath, array $options = []
         $dumpOptions->push(sprintf('-p%s', escapeshellarg($this->connectionConfig['password'])));
         $dumpOptions->push(sprintf('--max-allowed-packet=%s', escapeshellarg(config('protector.maxPacketLength'))));
         $dumpOptions->push('--no-create-db');
-        $dumpOptions->push('--set-gtid-purged=off');
+
+        if (! DB::connection()->isMaria()) {
+            $dumpOptions->push('--set-gtid-purged=off');
+        }
```

Comment:
looks great, while youre at it, can you write a line or two into the readme file, regarding the supported database software? preferrably near the top.

Reply:
Done.